src/main/resources/plugin-metadata/plugin-assembly.xml

```diff
@@ -9,25 +9,25 @@
         <fileSet>
             <directory>${project.basedir}/src/main/plugin-metadata</directory>
             <includes>
-              <include>plugin-security.policy</include>
+                <include>plugin-security.policy</include>
             </includes>
-            <outputDirectory></outputDirectory>
+            <outputDirectory>/elasticsearch</outputDirectory>
```

Comment:
Why the change?

Reply:
Won't work without it in new ES versions.
https://asciinema.org/a/LtxZyF5gAuoxX9cwFQvHsNoTQ